client/extensions/wp-job-manager/state/settings/test/actions.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+/**
+ * Internal dependencies
+ */
+import {
+	WP_JOB_MANAGER_FETCH_ERROR,
+	WP_JOB_MANAGER_FETCH_SETTINGS,
+	WP_JOB_MANAGER_UPDATE_SETTINGS,
+} from '../../action-types';
+import {
+	fetchError,
+	fetchSettings,
+	updateSettings,
+} from '../actions';
+
+describe( 'actions', () => {
+	const siteId = 123456;
+	const settings = {
+		data: {
```

Comment:
I suppose because it's a more accurate reflection of the structure of the response that the API sends back. 🤷‍♀️